### DISCUSSION TITLE
Any way to turn off all timeouts?

### DISCUSSION THREAD

**Discussion by macintacos** [ggandor/lightspeed.nvim#90] [Q&A] [answered]
I found the following settings:

```
exit_after_idle_msecs
jump_on_partial_input_safety_timeout
```

.. and while they have the desired affect of increasing timeouts in some modes (or removing timeouts entirely), I can't seem to find a way to turn off timeouts completely, such that if I hit "ESC" I'll be left wherever the cursor has currently navigated to (if we haven't reached a "final destination", so to speak). No matter what I tweak, if I hit `s` and type one character and think on what the next character is going to be, there's still a chance that lightspeed will timeout and I'll have to repeat the pattern.

Is there any way to turn off _all_ timeouts in lightspeed?

**Comment by ggandor** [comment, marked as answer]
https://github.com/ggandor/lightspeed.nvim/blob/153e6cb81b1360c67ec6c6138dc8df574264da5f/doc/lightspeed.txt#L142-L146

The value of `jump_on_partial_input_safety_timeout` is relevant only when jumping to a unique character automatically. (It specifies for how long should `b` as a keystroke be ignored after jumping to an `ab`, when `a` is the only one in the window, so we are jumping right after pressing `a`.)
